src/components/containers/Dialog/Dialog.module.css

```diff
@@ -1,11 +1,23 @@
 .container {
     display: flex;
-    box-sizing: border-box;
-    background-color: #FFFFFF;
+    flex-direction: column;
+    align-items: center;
+
     padding: 16px;
+    width: 100%;
+    max-width: 420px;
+
     border-radius: 16px;
-    flex-direction: column;
+
+    background-color: #FFFFFF;
+}
```

Comment:
idk if it's ok to center content, `display: block` is *more default* and easier to work with